src/pages/EmployeePage/EmployeePage.jsx

```diff
@@ -1,7 +1,12 @@
+/* eslint-disable react/jsx-no-undef */
+import ProfileInfo from '../../components/ProfileInfo/ProfileInfo';
+import InformatoionEmployee from '../../components/InfommatoionEmployee/InformatoionEmployee';
```

Comment:
```suggestion
import InformationEmployee from '../../components/InformationEmployee/InformationEmployee';
```
Хорошая привычка просматривать свои ПРы после открытия - часто находишь много вещей, оставленных по невнимательности.
Не ленитесь все делать аккуратно с первого раза - все неаккуратности рано или поздно выстрелят + эффект разбитых окон. Если нет времени все делать аккуратно с первого раза - то тем более не будет времени переделывать.